### ISSUE TITLE
Ignore specific instances of an Linter Error

### ISSUE THREAD

**Issue by mateusz-matyjaszczyk** [iljapostnovs/ui5plugin-linter#152]
Hey,

I would like to ask if it is possible to ignore specific instances of an Linter error.
I have following code:
![CleanShot 2024-09-20 at 17 43 42@2x](https://github.com/user-attachments/assets/319df5b6-297d-4263-9186-2bbe61a84e3a)
which is showing an error but actually I want to unbind context of an not named model so this is exactly what I want to do. I don't want to disable WrongParametersLinter for the whole project, I would just like to ignore it in this line. Is there anything I can do to achieve this?

**Comment by mateusz-matyjaszczyk** [comment]
@iljapostnovs

**Comment by iljapostnovs** [comment]
Hi,

This works as documented by SAP, but you can add this method to exceptions in `ui5` -> `ui5linter` un `package.json`:
![image](https://github.com/user-attachments/assets/9443b58a-6d5b-462b-b0c1-3ca43cc2f992)

**Comment by mateusz-matyjaszczyk** [comment]
Hey,
this method I am aware of. I don't want to do this as this will stop checking this method overall. What I am looking for is just ignore line below, something like @ts-ignore - i.e. ignoring next line with check, not all occurrences of it.

**Comment by iljapostnovs** [comment]
Yeah, there is no such feature currently. Probably will leave this as enhancement for the future.
